packages/location-management/src/ducks/location-units.ts

```diff
@@ -90,18 +90,8 @@ export interface LocationUnitPayloadPUT extends LocationUnitPayloadPOST {
 /** reducer name for the Item module */
 export const locationUnitsReducerName = 'location-units';
 
-/** Action types */
-const customfetchedActionType = 'location-units/LOCATION_UNITS_FETCHED';
-const customRemoveActionType = 'location-units/REMOVE_LOCATION_UNITS';
-const customSetTotalRecordsActionType = 'location-units/SET_TOTAL_LOCATION_UNITS';
-
 /** Item Reducer */
-export const locationUnitsReducer = reducerFactory<LocationUnit>(
-  locationUnitsReducerName,
-  customfetchedActionType,
-  customRemoveActionType,
-  customSetTotalRecordsActionType
-);
+export const locationUnitsReducer = reducerFactory<LocationUnit>(locationUnitsReducerName);
```

Comment:
Why did we remove the other params?

Reply:
https://github.com/OpenSRP/web/issues/190.

I have not looked into this yet, but it seems when you mount the component and dispatch the first time it does not use the correct action type.

Reply:
@p-netm Ok...noted